src/cli/cli.cpp

```diff
@@ -1897,7 +1897,6 @@ otError Interpreter::ProcessPartitionId(uint8_t aArgsLength, char *aArgs[])
         OutputLine("%u", otThreadGetPartitionId(mInstance));
         error = OT_ERROR_NONE;
     }
-#if OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE
```

Comment:
The  partitionid related macros`OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE` are added by the [CL](https://github.com/openthread/openthread/pull/5872/files). Should we remove all macros `OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE` added by the [CL](https://github.com/openthread/openthread/pull/5872/files).

Reply:
readme fixed